extensions/wikia/CommunityPage/styles/components/_avatar.scss

```diff
@@ -16,6 +20,14 @@
 		background-position: center;
 		background-repeat: no-repeat;
 	}
+
+	&.more {
+		background: $avatar-number-background;
```

Comment:
Talked to Ania. We established rules to tie it with wiki colors. Comming soon. Thanks for pointing that out.